XenModel/Messages.resx

```diff
@@ -14722,4 +14722,16 @@ Any disk in your VM's DVD drive will be ejected when installing {1}.</value>
   <data name="YOU_ARE_HERE" xml:space="preserve">
     <value>You are here</value>
   </data>
-</root>
+  <data name="FILE_NAME_IS_DEVICE_NAME_ERROR_MESSAGE" xml:space="preserve">

```

Comment:
- Please keep the entries in this file in alphabetical order.
- Stray "can not".